man/systemd-tmpfiles.xml

```diff
@@ -139,6 +139,14 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--purge-owned</option></term>
```

Comment:
to make this more symmetric to the other operations we could also just call this "purge", i.e. drop the "-owned" suffix?